pkg/windows/buildenv/salt-master.bat

```diff
@@ -1,11 +0,0 @@
-@ echo off
-:: Script for starting the Salt-Master
-:: Accepts all parameters that Salt-Master accepts
-
-:: Define Variables
-Set SaltDir=%~dp0
-Set SaltDir=%SaltDir:~0,-1%
-Set Python=%SaltDir%\bin\python.exe
-Set Script=%SaltDir%\bin\Scripts\salt-master
```

Comment:
Wouldn't salt-master.bat, and other related batch files be needed for the next release version of salt? I was hoping that the next version of salt would officially support Windows salt-master. We currently use them since we run salt-master on Windows, but we could always add them in even if they aren't in official salt. They do work.